library/core/src/time.rs

```diff
@@ -308,6 +308,30 @@ impl Duration {
         Duration { secs, nanos: subsec_nanos }
     }
 
+
+    /// Creates a new Duration from the specified number of nanoseconds.
+    /// 
+    /// Use this function if you need to specify time greater than what can fit in u64 
+    /// (around 584 years).
+    /// 
+    /// # Examples 
+    /// 
+    /// ```
+    /// use std::time::Duration;
```

Comment:
What @tgross35 meant was to add the feature gate `#![feature(duration_from_nanos_u128)]` _within_ this example.

Just for your understanding on why this is needed:
1. You're marking this function behind an `#[unstable(feature = ...)]` annotation. Think of it as the "lock" to this functionality.
2. Users can use this particular functionality only when they use the corresponding "key" in their codebase, which is the `#![feature(...)]` annotation.
3. This example in the comment is in reality a Rustdoc test which is used in the CI to test the usage of this functionality. So, to enable the testing for this functionality, you'll need to add the annotation _within_ this example.

So all you need to do is:

```suggestion
    /// #![feature(duration_from_nanos_u128)]
    /// use std::time::Duration;
```

Reply:
Hey thanks for the explanation, Will add this within the doc comment.